packages/vue-components/src/Retriever.vue

```diff
@@ -58,13 +58,21 @@ export default {
       if (this.hasFetchedCopy) {
         return;
       }
-      jQuery.get(this.srcWithoutHash)
-        .done((response) => {
-          let result = response;
+      fetch(this.srcWithoutHash)
+        .then(response => response.text())
+        .then((htmlText) => {
+          let result = htmlText;
           if (this.hash) {
-            const tempDom = jQuery('<temp>').append(jQuery.parseHTML(result));
-            const appContainer = jQuery(`#${this.hash}`, tempDom);
-            result = appContainer.html();
+            const htmlResult = document.implementation.createHTMLDocument('');
+            htmlResult.body.innerHTML = result;
+
+            // Script tags are removed to prevent Vue warnings about side effects in Vue template
+            // Ok to remove because Vue will not process the script tags to avoid side effects
+            const allScriptChildren = htmlResult.querySelectorAll('script');
+            allScriptChildren.forEach(child => child.remove());
+
+            const appContainer = htmlResult.querySelector(`#${this.hash}`);
```

Comment:
What if for some reason the selected element contains a script tag, will it be accidentally remove due to the above?
As in, are we sure that the appContainer will never contain perfectly valid script stuff? If not we should only try to remove the script injected by live-server in the code above

Reply:
I think the problem is that even if the user wants to put valid script inside, its not going to work because Vue does not process the script tag to avoid these side effects. 

Just tried to include this fragment 

```
<box type="info" id="markbind-action-tip">

MarkBind provides a helper GitHub Action that you can use to simplify your workflow file. A sample workflow file utilizing the action is as follows:
<script>
    alert("hi");
</script>
</box>
```

and the alert does not appear and the warning also appears in the console. So in that sense, its not gonna be processed even with these two lines.

Reply:
Changed to remove only the script in SVG tags in 1b9f5e8